src/storage/src/engine.rs

```diff
@@ -352,6 +352,18 @@ impl<S: LogStore> EngineInner<S> {
         })
     }
 
+    async fn close_region(&self, name: &str) -> Result<()> {
+        if let Some(RegionSlot::Ready(region)) =
+            self.regions.get_or_occupy_slot(name, RegionSlot::Opening)
```

Comment:
Why insert `RegionSlot::Opening` if the slot is not occupied when closing a region? Is `get_region` method more appropriate here?